src/lib/transform.js

```diff
@@ -70,8 +88,11 @@ export default (node, property, options, result) => {
         ? transformSelector(node[property], options, result, node)
         : transformValue(node[property], options, result, node);
   } catch (error) {
-    result.warn(error.message, { node });
-
+    if (error instanceof Error) {
```

Comment:
Is there a way to avoid this?

Reply:
Using `as`/`/** @type {Error} **/ (error)`